nb/3.fig1.hetionet_degree_dist/2.plot_hetionet_degree_distributions.ipynb

```diff
@@ -191,14 +193,14 @@
      "text": [
       "Warning message:\n",
       "“Transformation introduced infinite values in continuous y-axis”Warning message:\n",
-      "“Removed 46652 rows containing non-finite values (stat_ydensity).”Warning message:\n",
+      "“Removed 51668 rows containing non-finite values (stat_ydensity).”Warning message:\n",
       "“Transformation introduced infinite values in continuous y-axis”Warning message:\n",
-      "“Removed 46652 rows containing non-finite values (stat_ydensity).”"
+      "“Removed 51668 rows containing non-finite values (stat_ydensity).”"
```

Comment:
Possibly should just filter our zeros before so it's more of an explicit decision and not side effect of transformation